cms/djangoapps/maintenance/views.py

```diff
@@ -94,9 +132,21 @@ def render_response(self):
         """
         A short method to render_to_response that renders response.
         """
-        if self.request.is_ajax():
-            return JsonResponse(self.context)
-        return render_to_response(self.template, self.context)
+        context = self.context
+        template_name = self.template
+
+        try:
+            context, template_name = ForcePublishCourseRenderStarted().run_filter(
+                context=context, template_name=template_name
+            )
+        except ForcePublishCourseRenderStarted.RedirectToPage as exc:
+            response = HttpResponseRedirect(exc.redirect_to or reverse('home'))
+        else:
+            if self.request.is_ajax():
+                response = JsonResponse(context)
+            response = render_to_response(template_name, context)
```

Comment:
```suggestion
            if self.request.is_ajax():
                response = JsonResponse(context)
            else:
                response = render_to_response(template_name, context)
```
To maintain the previous behavior.

Reply:
Fixed on [fix: maintain force publish logic error](https://github.com/eduNEXT/edunext-platform/pull/776/commits/1cdce1fd5a4c337f20df79340ef4a7c4f1cb41a8)